spp_irrigation/__manifest__.py

```diff
@@ -15,6 +15,9 @@
         "base",
         "spp_base_gis",
     ],
+    "excludes": [
+        "spp_base",
+    ],
```

Comment:
@gonzalesedwin1123 i don't think that `spp_irrigation` should exclude `spp_base` since spp_irrigation module is just module to define a model and nothing else, meaning is that this module is just a skeletal module.

Reply:
@reichie020212 Is this not for farmer registry only?

Reply:
@gonzalesedwin1123 in the past only the farmer registry is using this module but it was removed so now there are no other modules that is using this module